docs/content/preview/releases/ybdb-releases/v2024.2.md

```diff
@@ -0,0 +1,476 @@
+
+### Downloads
+
+<ul class="nav yb-pills">
+  <li>
+    <a href="https://downloads.yugabyte.com/releases/2024.2.0.0/yugabyte-2024.2.0.0-b136-darwin-x86_64.tar.gz">
+      <i class="fa-brands fa-apple"></i>
+      <span>macOS</span>
+    </a>
+  </li>
+  <li>
+    <a href="https://downloads.yugabyte.com/releases/2024.2.0.0/yugabyte-2024.2.0.0-b136-linux-x86_64.tar.gz">
+      <i class="fa-brands fa-linux"></i>
+      <span>Linux x86</span>
+    </a>
+  </li>
+  <li>
+    <a href="https://downloads.yugabyte.com/releases/2024.2.0.0/yugabyte-2024.2.0.0-b136-el8-aarch64.tar.gz">
+      <i class="fa-brands fa-linux"></i>
+      <span>Linux ARM</span>
+    </a>
+  </li>
+</ul>
+
+**Docker:**
+
+```sh
+docker pull yugabytedb/yugabyte:2024.2.0.0-b136
+```
+
+### Highlights
+
+We're excited to announce the [Early Access](/preview/releases/versioning/#early-access-ea) and [General Availability](/preview/releases/versioning/#general-availability-ga) of the following powerful new features in YugabyteDB aimed at simplifying operations, enhancing functionality, and improving performance.
+
+**Yugabyte Kubernetes Operator** {{<tags/feature/ea>}}
+
+The [Yugabyte Kubernetes Operator](/preview/quick-start/kubernetes/#yugabytedb-kubernetes-operator) is a powerful tool designed to automate deploying, scaling, and managing YugabyteDB clusters in Kubernetes environments. It streamlines database operations, reducing manual effort for developers and operators. For more information, refer to the [YugabyteDB Kubernetes Operator](https://github.com/yugabyte/yugabyte-k8s-operator) GitHub project.
+
+**Active Session History** {{<tags/feature/ea>}}
+
+Get real-time and historical views of system activity by sampling session activity in the database. This feature can be used to troubleshoot performance issues effectively. For more information, refer to [Active Session History](/stable/explore/observability/active-session-history/) (ASH) documentation.
+
+**Instant Database Cloning** {{<tags/feature/ea>}}
+
+[Instant Database Cloning](/stable/manage/backup-restore/instant-db-cloning/) allows you to quickly create independent copies of your database for data recovery, development, and testing. Clones can be created in seconds, and initially consume no additional disk space because they share the same data files as the original database at creation but operate independently after that. Clones can be created as of now or at any time in the recent past (within a configurable history retention period), enabling developers to experiment without impacting production performance. This feature also provides a safety net for rapid recovery from accidental SQL or CQL human errors that cause data loss or corruption.
```

Comment:
```suggestion
[Instant database cloning](/stable/manage/backup-restore/instant-db-cloning/) allows you to quickly create independent copies of your database for data recovery, development, and testing. Clones can be created in seconds, and initially consume no additional disk space because they share the same data files as the original database at creation but operate independently after that. Clones can be created as of now or at any time in the recent past (within a configurable history retention period), enabling developers to experiment without impacting production performance. This feature also provides a safety net for rapid recovery from accidental SQL or CQL human errors that cause data loss or corruption.
```